internal/status/status.go

```diff
@@ -141,10 +141,11 @@ func (u *UpdateWriter) Send(update Update) {
 // isStatusEqual checks if two objects have equivalent status.
 //
 // Supported objects:
-//  GatewayClasses
-//  Gateway
-//  HTTPRoute
-//  TLSRoute
+//
+//	GatewayClasses
+//	Gateway
+//	HTTPRoute
+//	TLSRoute
```

Comment:
You must be running go 1.19 since gofmt now adds a space to doc comments. Since the project still uses 1.18, this change should be reverted.

Reply:
Ah yep that would be it. Switched to go 1.19 for some other work and didn't notice.